app/assets/stylesheets/application.scss

```diff
@@ -4,6 +4,8 @@ $govuk-new-typography-scale: true;
 $govuk-assets-path: "/";
 
 @import "govuk-frontend/dist/govuk/index";
+@import "@ministryofjustice/frontend/moj/components/sub-navigation/sub-navigation";
+@import "@ministryofjustice/frontend/moj/components/sortable-table/sortable-table";
```

Comment:
Does this resolve https://dsdmoj.atlassian.net/browse/AP-4433

Reply:
Yeah, I think so